### DISCUSSION TITLE
Assert object array equivalent

### DISCUSSION THREAD

**Discussion by Ams627** [fluentassertions/fluentassertions#1933] [General]
We used to be able to do this (5.9)

```cs
var arr = new object[] { 1, "Hello" };
arr.Should().BeEquivalentTo(1, "Hello");
```

What is the "modern" equivalent in 6.7?

**Comment by jnyrup** [comment]
For 6.0+ we removed the `BeEquivalentTo(params object[])` overload as it lead to ambiguities.
So you'll need to explicitly wrap `1` and `"Hello"` in some collection
```cs
var arr = new object[] { 1, "Hello" };
arr.Should().BeEquivalentTo(new object[] { 1, "Hello" });
```

See https://fluentassertions.com/upgradingtov6#collections for more details about the change.
